dir.props

```diff
@@ -196,6 +196,10 @@
     <TargetGroup Condition="'$(TargetGroup)'=='' and $(Configuration.Contains('wpa81'))">wpa81</TargetGroup>
     <TargetGroup Condition="'$(TargetGroup)'=='' and $(Configuration.Contains('uap101aot'))">uap101aot</TargetGroup>
     <TargetGroup Condition="'$(TargetGroup)'=='' and $(Configuration.Contains('uap101'))">uap101</TargetGroup>
+
+    <ProjectRunConfig Condition="'$(TargetGroup)'!=''">$(OSGroup).$(Platform).$(TargetGroup).$(ConfigurationGroup).$(TestTFM)</ProjectRunConfig>
```

Comment:
+1 to being against double dots.  It will always trigger the "is that a bug?" response for me.